nginx/server_error_on.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
```

Comment:
A third copy of this file?

Reply:
good catch thanks, I was confused by nginx vs hs_proxy and forgot to remove the extra files